webapp/channels/src/components/rhs_popout/rhs_popout.tsx

```diff
@@ -43,21 +50,33 @@ export default function RhsPopout() {
         }
     }, [dispatch, teamId]);
 
+    if (!channel || !team) {
+        return <LoadingScreen/>;
+    }
+
     return (
         <>
             <UnreadsStatusHandler/>
-            <div className='main-wrapper rhs-popout'>
+            {currentChannel && currentTeam && <div className='main-wrapper rhs-popout'>
```

Comment:
We need current team and channel set before `ChannelInfoRhs` is loaded or else it crashes.